### PR TITLE
add google_logging_metric to version 5 upgrade doc for #8780

### PR DESCRIPTION
**Pull request by pengq-google**
If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [X] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [X] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [X] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/develop/run-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read [Write release notes](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none
```

### CODE REVIEW

**Comment by pengq-google** [comment]
@c2thorn This is the major release doc change for #8780 
Thanks for your help along the way!

File: mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown [hunk 1]
```diff
@@ -331,3 +331,17 @@ If you were relying on accessing an individual flag by index (for example, `goog
 ### `rule.rate_limit_options.encorce_on_key` no longer has default value
 
 Previously, the default value for `rule.rate_limit_options.encorce_on_key` is "ALL", now this field no longer has a default value.
+
+## Resource: `google_logging_metric`
+
+### Additional fields are now required metrics with `value_type = "DISTRIBUTION"`
```
Comment (c2thorn) [inline mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown:337]:
```suggestion
### Additional `bucket_options` subfields are now properly required
```

Comment (c2thorn) [reply mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown:337]: as the PR is written, the `value_type` does not determine if the fields are required.

File: mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown [hunk 2]
```diff
@@ -331,3 +331,17 @@ If you were relying on accessing an individual flag by index (for example, `goog
...
+
+For log-based metrics with `value_type = "DISTRIBUTION"`, additional fields are now required depending on the `bucket_options` block:
```
Comment (c2thorn) [inline mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown:339]:
```suggestion
When setting the `bucket_options` block, the following fields may be required:
```

File: mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown [hunk 3]
```diff
@@ -331,3 +331,17 @@ If you were relying on accessing an individual flag by index (for example, `goog
...
+
+* `num_finite_buckets`, `width`, and `offset` are now required when `bucket_options` is set to be `linear_buckets`
+
+* `num_finite_buckets`, `growth_factor`, and `scale` are now required when `bucket_options` is set to be `exponential_buckets`
+
+* `bounds` is now required when `bucket_options` is set to be `explicit_buckets`.
```
Comment (c2thorn) [inline mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown:345]:
```suggestion
* `num_finite_buckets`, `width`, and `offset` are now required when `bucket_options.linear_buckets` is set.

* `num_finite_buckets`, `growth_factor`, and `scale` are now required when `bucket_options.exponential_buckets` is set.
```
the `bounds` requirement was not added in the 5.0.0 PR?

**Review by c2thorn — CHANGES_REQUESTED** [review]

**Comment by c2thorn** [comment]
I believe this has failed because there are now conflicts @pengq-google

**Comment by pengq-google** [comment]
replaced by https://github.com/GoogleCloudPlatform/magic-modules/pull/8886

**Pull request closed without merging**